Makefile

```diff
@@ -18,7 +15,7 @@ bin/kubebuilder:
 	hack/download-kubebuilder-local.sh
 
 # Run tests
-test: generate fmt vet manifests bin/kubebuilder
+test: generate fmt vet verify-manifests bin/kubebuilder
```

Comment:
What about other places we depend on `manifests`. Like the `kind` target?

Reply:
I've updated the branch with some further verifications.

And added a `verify` target which runs them all.